index.html

```diff
@@ -143,89 +147,212 @@ <h1 id="logo"><a href="index.html">GreatUniHack</a></h1>
+                                            <td>10:00</td>
+                                            <td><i>Workshop:</i> CV</td>
+                                        </tr>
+                                        <tr>
+                                            <td>11:30</td>
+                                            <td>Hacking Ends, Lunch</td>
+                                        </tr>
+                                        <tr>
+                                            <td>12:00</td>
+                                            <td>Lunch</td>
+                                        </tr>
+                                        <tr>
+                                            <td>12:30</td>
+                                            <td>Judging</td>
+                                        </tr>
+                                        <tr>
+                                            <td>14:00</td>
+                                            <td>Closing Ceremony</td>
+                                        </tr>
+                                        <tr>
+                                            <td>16:00</td>
+                                            <td>End of the Event</td>
+                                        </tr>
+                                    </tbody>
+                                </table>
                             </div>
                         </div>
                     </div>
+                </div>
+            </section>
+            
 
-                    <h3>Gold</h3>
-                    <div class="box alt">
-                        <div class="row 50% uniform">
-                            <div class="6u 12u(xsmall)">
-                                <a href="https://www.webapplicationsuk.com" class="image fit">
-                                    <img src="images/webapps.png" alt="Web Applications UK" />
-                                </a>
-                            </div>
-                            <div class="6u 12u(xsmall)">
-                                <a href="https://ocadotechnology.com" class="image fit">
-                                    <img src="images/ocado-technology.png" alt="Ocado Technology" />
-                                </a>
-                            </div>
-                            <div class="12u 12u(xsmall)">
-                                <a href="https://www.goldmansachs.com" class="image fit">
-                                    <img src="images/goldman-sachs.png" alt="Goldman Sachs" />
-                                </a>
+            <!-- Sponsors
+                 For mobile screens, if we want pictures one below another, use 12u(xsmall)
+                 If we want 2 pictures per line, use 6u
+                 If we want 3 pictures per line, use 3u -->
+            <section id="sponsors" class="wrapper style1 special fade-up">
+                <div style="background-color:#272833; width=100%">

```

Comment:
We should not have styles only for this div in the html, that is why we have a stylesheet. Please use the class of this section and add the background attribute to it. Please use assests/css/main.css for reference, especially the last part of the file.